iOS/Flows/Conversation/ConversationViewController.swift

```diff
@@ -201,22 +201,11 @@ class ConversationViewController: FullScreenViewController,
         return IndexPath(item: index, section: 0)
     }
     
-    // MARK: - UICollectionViewDelegate
-    
-    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        guard let messageItem = self.dataSource.itemIdentifier(for: indexPath) else { return }
-        
-        switch messageItem {
-        case .message(let messageID):
-            self.onSelectedThread?(self.conversation.cid, messageID)
-        case .loadMore:
-            self.loadMoreMessageIfNeeded()
-        }
-    }
```

Comment:
With a collection view inside the cells, this delegate function no longer gets called.  Setting up the input handlers up above gets around this issue.